README.md

```diff
@@ -1,77 +1,64 @@
+## Block Layer (Layer 0)
 
-Many of the specifications in this stack are inter-dependent.
+The block layer encompasses all content addressed block formats and specifies how blocks
+are addressed,how they self-describe their codec for encoding/decoding, and how block link
+between each other.
 
-```
-                          IPLD Dependency Graph
-
-┌─────┐                           ┌────────────┐
-│ CID ├───────────┬───────────────> Raw Blocks │
-└─────┘           │               └─┬──────────┘
-          ┌───────v──────────────┐  │
-┌──────┐  │ Links (Conceptually) │  │
-│ Path │  └───────┬──────────────┘  │           ┌─────────────┐
-└──┬───┘          │                 ├───────────> Replication │
-   │              │                 │           └─────────────┘
-┌──v──────────────v─────────────────┴──────┐
-│                  Codecs                  │
-│  ┌─────┐   ┌───────────────────────────┐ │ ┌────────────────────────────┐
-│  │ Git │   │       Data Model v1       │ │ │  Multi-block Collections   │
-│  └─────┘   │ ┌──────────┐ ┌──────────┐ │ │ │ ┌────────────────────────┐ │
-│ ┌────────┐ │ │ dag-json │ │ dag-cbor │ ├─┼─> │    Specialized Data    │ │
-│ │  BTC   │ │ └──────────┘ └──────────┘ │ │ │ │ Structures & Utilities │ │
-│ └────────┘ └───────────────────────────┘ │ │ └────────────────────────┘ │
-│ ┌────────┐ ┌────────┐                    │ └───┬─────────┬──────────────┘
-│ │ Zcash  │ │ dag-pb │                    │     │         │
-│ └────────┘ └────────┘                    │     │         │
-└──────────────────────────────────────────┘     │         │
-                                                 │         │
-               ┌──────────────────────┐          │         │
-               │ File System (UnixFS) <──────────┘         │
-               └──────────────────────┘                    │
-               ┌───────────────────────┐                   │
-               │ End-User Applications <───────────────────┘
-               └───────────────────────┘
+This layer alone is enough to accomplish basic graph replication across a variety of formats (eth, bitcoin, git, etc).
 
-```
+This layer does not define data structures or types, although many codecs may convert these formats into native types,
+there are no type requirements or assurances about types at the block layer.
+
+## Data Model Layer (Layer 1)
+
+The Data Model Layer describes a set of base required types to be implemented by a subset of IPLD codecs.
+
+With these basic types authors can create various single-block data structures which can be read with predictable
+paths and selectors.
+
+With just the data model layer, several data structures can be authored and put into a single block. These data 
+structures can also link to one another, but a *single* data structures's namespace cannot be spread across many
+blocks. This limitation means that collections authored in only the Data Model layer are limited by the maximum
+block size (typically 2 megabytes).
```

Comment:
I'd split this mention of block size limits into its own paragraph, and try to phrase it as *some* transports having block size limits.

Perhaps some text to the effect that "ideally, a system designs to produce blocks of limited size: since a block is the unit of data that must be loaded into memory at once for any operation, a good size is on the order of megabytes.  larger datastructures should be sharded (and advanced data layout features such as HAMT, etc, can be used do this transparently)."   And then we can mention the example of libp2p's practical size limits as an instance of why this is good to mind.